### PR TITLE
Static from web component. Monorepo.

### PR DESCRIPTION
**Pull request by sokolovstas**

### CODE REVIEW

**Comment by erudenko** [comment]
My biggest question is, what is an advantage of `goth` compare to the default oath2 package [https://pkg.go.dev/golang.org/x/oauth2](https://pkg.go.dev/golang.org/x/oauth2)?

Default package has a list of options:
CleanShot 2021-07-27 at 17 25 35@2x

**Comment by sokolovstas** [comment]
> My biggest question is, what is an advantage of `goth` compare to the default oath2 package

Goth wrap oauth2 with default prams for concrete provider and allow to use common interface:
GetAuthURL() -> GetUser()

Goth doesn't do any magic, we can just copy and paste all the code from Goth, but why?

**Comment by erudenko** [comment]
@sokolovstas do we need `Circe-Bold` font? It is not a free font and requires a licence. We need to exclude it from the public version.

**Comment by sokolovstas** [comment]
> do we need `Circe-Bold` font? It is not a free font and requires a licence. We need to exclude it from the public version.

This font was in Admin UI. Is there any replacement?

File: cmd/config-boltdb.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 general: # General server settings.
+  port: 8081
```
[the remark this answers is not in the document]
Comment (sokolovstas) [reply cmd/config-boltdb.yaml:2]: Yes but for docker we can have host `identifo.com:443` but serve it on `identifo-service:8081` behind nginx proxy.

**Comment by sokolovstas** [comment]
> @sokolovstas do we need `Circe-Bold` font? It is not a free font and requires a licence. We need to exclude it from the public version.

Fonts replaced

**Pull request merged**